### PR TITLE
feat(ifr): support Molmo-72b

### PR DESCRIPTION
**Pull request by tgenaitay**
### Your checklist for this pull request

- [X] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

- Described Molmo-72B for Managed Inference
- Edited nav
- [x] Test function calling and document support accordingly
- [x] Test limitations in file size, file types, file volume
- [x] Test code snippets

### CODE REVIEW

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+## Compatible Instances
+
+| Instance type  | Max context length |
+| ------------- |-------------|
+| H100-2      | 50k (FP8)
+
+## Model introduction
+
+Molmo 72B is the powerhouse of the Molmo family, multimodal models developed by the renowned research lab Allen Institute for AI.
+Vision-language model like Molmo can analyze images and offer insights from visual content alongside text. This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension.
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:40]:
```suggestion
Vision-language models like Molmo can analyze images and offer insights from visual content alongside text. This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension. 
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+| Instance type  | Max context length |
+| ------------- |-------------|
+| H100-2      | 50k (FP8)
+
+## Model introduction
+
+Molmo 72B is the powerhouse of the Molmo family, multimodal models developed by the renowned research lab Allen Institute for AI.
+Vision-language model like Molmo can analyze images and offer insights from visual content alongside text. This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension. 
+
+Molmo is open-weight and distributed under the Apache 2.0 license. All artifacts (code, data set, evaluations) are also expected to be full open-source.
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:42]:
```suggestion
Molmo is open-weight and distributed under the Apache 2.0 license. All artifacts (code, data set, evaluations) are also expected to be fully open-source.
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+Its base model is Qwen2-72B ((Twonyi Qianwen license)[https://huggingface.co/Qwen/Qwen2-72B/blob/main/LICENSE]).
+ 
+## Why is it useful?
+
+- Molmo-72b allows you to process real world and high resolution images, unlocking capacities such as transcribing handwritten files or payment receipts, extracting information from graphs, captioning images, etc.
+- This model achieves the [highest academic benchmark scores and second rank on human evaluations](https://huggingface.co/allenai/Molmo-72B-0924#evaluations) as of writing (September 2024)
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:48]:
```suggestion
- This model achieves the [highest academic benchmark scores and ranks second on human evaluation](https://huggingface.co/allenai/Molmo-72B-0924#evaluations) at the time of writing (September 2024)
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  The model name allows Scaleway to put your prompts in the expected format.
+</Message>
+
+<Message type="note">
+  Ensure that the `messages` array is properly formatted with roles (system, user, assistant) and content.
+</Message>
+
+### Passing images to Molmo-72b
+
+1. Image URLs
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:105]:
```suggestion
#### Image URLs
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+If the image is available online, you can just include the image URL in your request as demonstrated above. This approach is simple and does not require any encoding.
+
+2. Base64 encoded image
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:108]:
```suggestion
#### Base64 encoded image
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+  Despite efforts for accuracy, the possibility of generated text containing inaccuracies or [hallucinations](/ai-data/managed-inference/concepts/#hallucinations) exists. Always verify the content generated independently.
+</Message>
+
+## Frequently Asked Questions
+
+#### What types of images are supported by Molmo-72b?
+- Bitmap (or raster) image formats, meaning storing images as grids of individual pixels, are supported: PNG, JPEG, WEBP, and non-animated GIFs in particular.
+- Vector image formats (SVG, PSD) are not supported.
+
+#### Are other files supported?
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:161]:
```suggestion
#### Are other file types supported?
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+Only bitmaps can be analyzed by Molmo, PDFs and videos are not supported.
+
+#### Is there a limit to the size of each image?
+The only limitation is in context window (1 token for each 16x16 pixel).
+
+#### What is the maximum amount of images per conversation?
+One conversation can handle up to 12 images (per request). The 13rd will return a 413 error.
```
Comment (RoRoJ) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:168]:
```suggestion
One conversation can handle up to 12 images (per request). The 13th will return a 413 error.
```

**Review by bene2k1 — APPROVED** [review]

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 8]
```diff
@@ -0,0 +1,165 @@
+---
+meta:
+  title: Understanding the Molmo-72b-0924 model
+  description: Deploy your own secure Molmo-72b-0924 model with Scaleway Managed Inference. Privacy-focused, fully managed.
+content:
+  h1:  Understanding the Molmo-72b-0924 model
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:6]:
```suggestion
  h1: Understanding the Molmo-72b-0924 model
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+  paragraph: This page provides information on the Molmo-72b-0924 model
+tags:
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:8]:
```suggestion
tags: ai molmo inference
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+| ------------- |-------------|
+| H100-2      | 50k (FP8)
+
+## Model introduction
+
+Molmo 72B is the powerhouse of the Molmo family, multimodal models developed by the renowned research lab Allen Institute for AI.
+Vision-language model like Molmo can analyze an image and offer insights from visual content alongside text. This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension. 
+
+Molmo is open-weight and distributed under the Apache 2.0 license. All artifacts (code, data set, evaluations) are also expected to be fully open-source.
+Its base model is Qwen2-72B ((Twonyi Qianwen license)[https://huggingface.co/Qwen/Qwen2-72B/blob/main/LICENSE]).
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:43]:
```suggestion
Its base model is Qwen2-72B ([Twonyi Qianwen license](https://huggingface.co/Qwen/Qwen2-72B/blob/main/LICENSE)).
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+- Molmo-72b allows you to process real world and high resolution images, unlocking capacities such as transcribing handwritten files or payment receipts, extracting information from graphs, captioning images, etc.
+- This model achieves the [highest academic benchmark scores and ranks second on human evaluation](https://huggingface.co/allenai/Molmo-72B-0924#evaluations) at the time of writing (September 2024)
+
+<Message type="note">
+  Molmo-72b can understand and analyze images, not generate them. You will use it through the /v1/chat/completions endpoint.
+</Message>
+
+## How to use it
+
+### Sending Inference requests
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:56]:
```suggestion
### Sending inference requests
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+<Message type="tip">
+  The model name allows Scaleway to put your prompts in the expected format.
+</Message>
+
+### Known limitations
+
+- Molmo-72b was reported to struggle with transparent images. The official recommandation is to add white or dark background to images for the time being.
+- Molmo-72b chat template doesn't support the system role. Ensure that the `messages` array is properly formatted with user role and assistant role only.
+- Molmo-72b isn't able to generate structured outputs (`response_format` parameter not supported)
+- Molmo-72b can't do function calling (`tools` parameter not supported)
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:98]:
```suggestion
- Molmo-72b was reported to struggle with transparent images. The official recommandation is to add white or dark background to images for the time being.
- Molmo-72b chat template does not support the system role. Ensure that the `messages` array is properly formatted with user role and assistant role only.
- Molmo-72b is not able to generate structured outputs (`response_format` parameter not supported).
- Molmo-72b cannot do function calling (`tools` parameter not supported).
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+</Message>
+
+## Frequently Asked Questions
+
+#### What types of images are supported by Molmo-72b?
+- Bitmap (or raster) image formats, meaning storing images as grids of individual pixels, are supported: PNG, JPEG, WEBP, and non-animated GIFs in particular.
+- Vector image formats (SVG, PSD) are not supported.
+
+#### Are other file types supported?
+Only bitmaps can be analyzed by Molmo, PDFs and videos are not supported.
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:159]:
```suggestion
Only bitmaps can be analyzed by Molmo. PDFs and videos are not supported.
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 14]
```diff
@@ -0,0 +1,165 @@
...
+
+#### Is there a limit to the size of each image?
+The only limitation is in context window (1 token for each 16x16 pixel).
+
+#### What is the maximum amount of images per conversation?
+One conversation can handle maximum 1 image (per request). Sending more than one image will return a 400 error.
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:165]:
```suggestion
One conversation can handle a maximum of 1 image (per request). Sending more than one image will return a 400 error.
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 15]
```diff
@@ -0,0 +1,165 @@
...
+
+#### What types of images are supported by Molmo-72b?
+- Bitmap (or raster) image formats, meaning storing images as grids of individual pixels, are supported: PNG, JPEG, WEBP, and non-animated GIFs in particular.
+- Vector image formats (SVG, PSD) are not supported.
+
+#### Are other file types supported?
+Only bitmaps can be analyzed by Molmo, PDFs and videos are not supported.
+
+#### Is there a limit to the size of each image?
+The only limitation is in context window (1 token for each 16x16 pixel).
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:162]:
```suggestion
The only limitation is the context window (1 token for each 16x16 pixel).
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 16]
```diff
@@ -0,0 +1,165 @@
...
+### Passing images to Molmo-72b
+
+#### Image URLs
+If the image is available online, you can just include the image URL in your request as demonstrated above. This approach is simple and does not require any encoding.
+
+#### Base64 encoded image
+Base64 encoding is a standard way to transform binary data, like images, into a text format, making it easier to transmit over the internet.
+
+The following Python code sample shows you how to encode an image in base64 format and pass it to your request payload.
+
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:109]:
```suggestion
```

File: ai-data/managed-inference/reference-content/molmo-72b-0924.mdx [hunk 17]
```diff
@@ -0,0 +1,165 @@
...
+## Compatible Instances
+
+| Instance type  | Max context length |
+| ------------- |-------------|
+| H100-2      | 50k (FP8)
+
+## Model introduction
+
+Molmo 72B is the powerhouse of the Molmo family, multimodal models developed by the renowned research lab Allen Institute for AI.
+Vision-language model like Molmo can analyze an image and offer insights from visual content alongside text. This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension.
```
Comment (jcirinosclwy) [inline ai-data/managed-inference/reference-content/molmo-72b-0924.mdx:40]:
```suggestion
Vision-language models like Molmo can analyze an image and offer insights from visual content alongside text. This multimodal functionality creates new opportunities for applications that need both visual and textual comprehension. 
```

**Review by jcirinosclwy — APPROVED** [review]

**Comment by tgenaitay** [comment]
Thanks for your reviews!

**Pull request merged**